gym/lib/gym/runner.rb

```diff
@@ -162,6 +167,23 @@ def copy_mac_app
       app_path
     end
 
+    # Make the .pkg from .app
+    def make_mac_pkg(app_path)
+      sign = Gym.config[:installer_codesigning_identity].shellescape
+      UI.crash!("Couldn't find Developer Installer Code Signing Identity") unless sign
```

Comment:
What are we checking here? This would already crash when we try to call `shellescape` on `nil`, right?